test/integration/dss/test_dss_api_retry.py

```diff
@@ -15,13 +15,13 @@
 pkg_root = os.path.abspath(os.path.join(os.path.dirname(__file__), '..', '..', '..'))  # noqa
 sys.path.insert(0, pkg_root)  # noqa
 
-import hca.dss
-from hca.util import RetryPolicy
-from hca.dss import upload_to_cloud
+import dbio.dss
+from dbio.util import RetryPolicy
+from dbio.dss import upload_to_cloud
 
 
 class TestDssApiRetry(unittest.TestCase):
-    staging_bucket = "org-humancellatlas-dss-cli-test"
+    staging_bucket = "ucsc-cgp-dss-cli-test"
```

Comment:
was this bucket created?

Reply:
No. My plan was to add this bucket name to the data store environment file first, then use terraform to create the bucket.

Reply:
Earlier, this issue came up (with auth URL, I think): do we want the data store environment variables to be used/required by the data store CLI?

If so, we can use the environment variable from the data store environment file to get this bucket name, and assume that it has already been created as part of the data store deployment process.

If not, the buckets will need to be created manually, and we should update the Readme/quick start with instructions that mention any test buckets that need to be created (& what name they should have).

Reply:
I think it would be OK to assume that the bucket would be built out as part of the deployment process within the data-store repo. 

Perhaps we can add a simple integration test that checks if the bucket exists to allow the CI to indicate a missing bucket.

Reply:
Will address this in a later PR

See DataBiosphere/data-store#49 for test bucket creation

See DataBiosphere/data-store-cli#21 for a test to check that the bucket exists

Reply:
Addressed by DataBiosphere/data-store#65